### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rdacdx**
Include examples for user settings use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,14 @@ Thanks to **[Tobias Nyholm](https://github.com/Nyholm)** and [Artem Zhuravlov](h
     // Each of methods above has last optional $user parameter
     // that allows to get/set per-user settings
     // Your User Entity must implement SettingsOwnerInterface if you wish to use per-user settings
+    // Example: class User extends BaseUser implements SettingsOwnerInterface
+    //
+    // You need to implement the getSettingIdentifier()
+    // Example:
+    // public function getSettingIdentifier()
+    // {
+    //     return $this->id;
```
Comment (Nyholm) [inline README.md]: Prefix this with 'user_'. This is good when you have multiple objects implementing the SettingsOwnerInterface.

Comment (rdacdx) [inline README.md]: Excuse me, I did not understood what did you mean.
Could you please provide to me a real example?
Thanks.

Comment (Nyholm) [inline README.md]: Sorry. I believe that you should have an identifier like: 
```php
return 'user_'.$this->id;
```

When prefixing with a string you avoid collisions if you have multiple entities in your application that could own a setting. 
```php
class Company implements SettingsOwnerInterface {
  public function getSettingIdentifier() {
    return 'cmpy_'.$this->id;
  }
}
```

**Comment by dmishh** [comment]
Made this more clear in #43 
Thanks @rdacdx @Nyholm :+1:

**Pull request closed without merging**